tesp/workflow.py

```diff
@@ -193,12 +195,19 @@ def run(self):
         fmask_doc_fname = Path(self.input()['fmask']['metadata'].path)
         gqa_doc_fname = Path(self.input()['gqa'].path)
 
+        tesp_doc_fname = Path(
+            pjoin(self.workdir, '{}.tesp.yaml'.format(self.granule))
+        )
+        with open(tesp_doc_fname, 'w') as src:
+            yaml.dump(_get_tesp_metadata(), src)
```

Comment:
Using pathlib but none of its methods :)

(and safe_dump is recommended, even though it doesn't really matter in this case)

```suggestion
        tesp_doc_fname = Path(self.workdir) / '{}.tesp.yaml'.format(self.granule)
        with tesp_doc_fname.open('w') as src:
            yaml.safe_dump(_get_tesp_metadata(), src)
```

Reply:
In a future update we'll migrate to pathlib.  Luigi might've updated to include Path as a type, but for the version we're using they're plain strings.

I'll use your suggestion.